<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="F:\INTECH\SEMESTRE 3\PI\BirdHouse-Battle\BirdHouse_Battle\BirdHouse_Battle.UnitTests\bin\Debug\net461\BirdHouse_Battle.UnitTests.dll" total="40" errors="0" failures="5" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2018-10-31" time="20:10:29">
  <environment nunit-version="2.6.4.14350" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 6.2.9200.0" platform="Win32NT" cwd="F:\INTECH\SEMESTRE 3\PI\BirdHouse-Battle\BirdHouse_Battle\BirdHouse_Battle.UnitTests\bin\Debug\net461" machine-name="LAPTOP-CK9E0OFE" user="aef52" user-domain="LAPTOP-CK9E0OFE" />
  <culture-info current-culture="fr-FR" current-uiculture="fr-FR" />
  <test-suite type="Assembly" name="F:\INTECH\SEMESTRE 3\PI\BirdHouse-Battle\BirdHouse_Battle\BirdHouse_Battle.UnitTests\bin\Debug\net461\BirdHouse_Battle.UnitTests.dll" executed="True" result="Failure" success="False" time="1.282" asserts="0">
    <results>
      <test-suite type="Namespace" name="BirdHouse_Battle" executed="True" result="Failure" success="False" time="1.273" asserts="0">
        <results>
          <test-suite type="Namespace" name="UnitTests" executed="True" result="Failure" success="False" time="1.273" asserts="0">
            <results>
              <test-suite type="TestFixture" name="ArenaTests" executed="True" result="Success" success="True" time="0.301" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.Aquire_a_target" executed="True" result="Success" success="True" time="0.078" asserts="2" />
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.create_team" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.remove_team" executed="True" result="Success" success="True" time="0.073" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.Spawn_unit" executed="True" result="Success" success="True" time="0.003" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.specific_spawn_location" executed="True" result="Success" success="True" time="0.121" asserts="4" />
                  <test-case name="BirdHouse_Battle.UnitTests.ArenaTests.unit_cannot_move_on_a_already_used_location" executed="True" result="Success" success="True" time="0.001" asserts="2" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="GameScreenTests" executed="True" result="Success" success="True" time="0.001" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.GameScreenTests.window_is_displayed" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="TeamTests" executed="True" result="Failure" success="False" time="0.347" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.add_a_goblin" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.add_a_paladin" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.add_an_archer" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.archer_count_is_exact" executed="True" result="Success" success="True" time="0.003" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.gold_calculation" executed="True" result="Failure" success="False" time="0.077" asserts="2">
                    <failure>
                      <message><![CDATA[  Expected: 25.0d
  But was:  45.0d
]]></message>
                      <stack-trace><![CDATA[à BirdHouse_Battle.UnitTests.TeamTests.gold_calculation()
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.remove_unit_from_team" executed="True" result="Success" success="True" time="0.001" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.team_is_wiped_when_there_is_no_unit_left" executed="True" result="Success" success="True" time="0.002" asserts="5" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.team_limit_is_effective" executed="True" result="Success" success="True" time="0.094" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.TeamTests.unit_count_is_exact" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="UnitTests_Archer" executed="True" result="Success" success="True" time="0.044" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Archer.Try_Die" executed="True" result="Success" success="True" time="0.002" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Archer.Try_InRange" executed="True" result="Success" success="True" time="0.002" asserts="42" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Archer.Try_New_Direction" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Archer.Try_Take_Damages" executed="True" result="Success" success="True" time="0.032" asserts="5" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="UnitTests_Gobelin" executed="True" result="Success" success="True" time="0.041" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Gobelin.Try_Die" executed="True" result="Success" success="True" time="0.001" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Gobelin.Try_InRange" executed="True" result="Success" success="True" time="0.001" asserts="42" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Gobelin.Try_New_Direction" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Gobelin.Try_Take_Damages" executed="True" result="Success" success="True" time="0.032" asserts="5" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="UnitTests_Paladin" executed="True" result="Success" success="True" time="0.043" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Paladin.Try_Die" executed="True" result="Success" success="True" time="0.001" asserts="3" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Paladin.Try_InRange" executed="True" result="Success" success="True" time="0.001" asserts="42" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Paladin.Try_New_Direction" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Paladin.Try_Take_Damages" executed="True" result="Success" success="True" time="0.034" asserts="5" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="UnitTests_Unit" executed="True" result="Failure" success="False" time="0.337" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Die_Null_Context" executed="True" result="Success" success="True" time="0.000" asserts="6" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Distance_Error" executed="True" result="Success" success="True" time="0.025" asserts="0" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Nearest_Enemy" executed="True" result="Success" success="True" time="0.001" asserts="4" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Diagonal_Negative" executed="True" result="Failure" success="False" time="0.067" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: not BirdHouse_Battle.Model.Vector
  But was:  BirdHouse_Battle.Model.Vector
]]></message>
                      <stack-trace><![CDATA[à BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Diagonal_Negative()
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Diagonal_Positive" executed="True" result="Failure" success="False" time="0.068" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: not BirdHouse_Battle.Model.Vector
  But was:  BirdHouse_Battle.Model.Vector
]]></message>
                      <stack-trace><![CDATA[à BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Diagonal_Positive()
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Horizon" executed="True" result="Failure" success="False" time="0.072" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: not BirdHouse_Battle.Model.Vector
  But was:  BirdHouse_Battle.Model.Vector
]]></message>
                      <stack-trace><![CDATA[à BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Horizon()
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Vertical" executed="True" result="Failure" success="False" time="0.068" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: not BirdHouse_Battle.Model.Vector
  But was:  BirdHouse_Battle.Model.Vector
]]></message>
                      <stack-trace><![CDATA[à BirdHouse_Battle.UnitTests.UnitTests_Unit.Try_Update_Vertical()
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="UnitTests_Vector" executed="True" result="Success" success="True" time="0.014" asserts="0">
                <results>
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Vector.Add_Test" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Vector.Effect_Var_To_X_And_Y" executed="True" result="Success" success="True" time="0.001" asserts="10" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Vector.Magnitude_Test" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Vector.Mutiply_Test" executed="True" result="Success" success="True" time="0.001" asserts="2" />
                  <test-case name="BirdHouse_Battle.UnitTests.UnitTests_Vector.Soustract_Test" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>